packages/forms-web-app/tests/unit/controllers/eligibility/decision-date.test.js

```diff
@@ -269,6 +268,7 @@ describe('controllers/eligibility/decision-date', () => {
       appeal.decisionDate = '2020-10-10';
 
       const date = addWeeks(endOfDay(parseISO(appeal.decisionDate)), 12);
+      expect(dateFilter(date, 'DD MMMM YYYY')).toBe('02 January 2021');
```

Comment:
double check that they want the leading zero - everywhere I've seen has been without the leading zero, but I don't know for sure

Reply:
Good question, I don't know

Reply:
> double check that they want the leading zero - everywhere I've seen has been without the leading zero, but I don't know for sure

Yes